bootloader/grub.go

```diff
@@ -379,11 +384,79 @@ func (g *grub) ManagedAssets() []string {
 	}
 }
 
-// CommandLine returns the kernel command line composed of the built-in
-// list and extra arguments passed in arguments. The command line may be
-// different when using a bootloader in the recovery partition.
+// CommandLine returns the kernel command line composed of system run mode
+// arguments, extra arguments (configured by snapd) and the built-in bootloader
```

Comment:
see previous comment here, I think it's more clear to not use the word extra here for the kernel params coming from inside snapd